scripts/dev

```diff
@@ -471,6 +472,12 @@ task :lint do
   sh "pre-commit run -a"
 end
 
+desc "Generate a new migration with the current utc timestamp prefix"
+task genmigration: :consume_args do |t, args|
+  migrationName = Time.now.utc.strftime("v%Y%m%d%H%M%S__.sql")
```

Comment:
Not sure if flyway actually cares or not, but for consistency can you make this a capital V?

```suggestion
  migrationName = Time.now.utc.strftime("V%Y%m%d%H%M%S__.sql")
```

Reply:
Done